src/NServiceBus.Metrics/ProbeBuilders/RetriesProbeBuilder.cs

```diff
@@ -16,8 +16,10 @@ public RetriesProbeBuilder(FeatureConfigurationContext context)
         protected override void WireUp(SignalProbe probe)
         {
             var errors = notifications.Errors;
+#pragma warning disable CS0618 // Type or member is obsolete
             errors.MessageHasFailedAnImmediateRetryAttempt += (sender, message) => Signal(message.Headers, probe);
             errors.MessageHasBeenSentToDelayedRetries += (sender, message) => Signal(message.Headers, probe);
+#pragma warning restore CS0618 // Type or member is obsolete
```

Comment:
Just guessing, is that stuff obsolete in 7.7 becuase it's getting cancellation tokens in V8? (which we obviously can't update it to yet in this version)